backend/src/main/scala/com/softwaremill/adopttapir/starter/api/StarterRequest.scala

```diff
@@ -11,7 +11,8 @@ case class StarterRequest(
     addDocumentation: Boolean,
     addMetrics: Boolean,
     json: JsonImplementationRequest,
-    scalaVersion: ScalaVersionRequest
+    scalaVersion: ScalaVersionRequest,
+    builder: BuilderRequest = BuilderRequest.Sbt
```

Comment:
I am aware that this is needed for backward compatibility. 

Please add at least TODO: to not forget to remove it during providing UI changes.

Reply:
Added